src/transformers/configuration_utils.py

```diff
@@ -317,6 +317,9 @@ def from_pretrained(cls, pretrained_model_name_or_path: Union[str, os.PathLike],
             proxies (:obj:`Dict[str, str]`, `optional`):
                 A dictionary of proxy servers to use by protocol or endpoint, e.g., :obj:`{'http': 'foo.bar:3128',
                 'http://hostname': 'foo.bar:4012'}.` The proxies are used on each request.
+            use_auth_token (:obj:`str` or `bool`, `optional`):
```

Comment:
Is it worth it to also allow `str` here? not sure if this could lead to weird bug like `"true"` works but `"True"` doesn't